src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -756,11 +756,11 @@ OverlayImpl::onManifests (
                 protocol::TMManifests o;
                 o.add_list ()->set_stobject (s);
 
-                std::set<Peer::id_t> skip;
-                hashRouter.prepRelay (hash, skip);
-                foreach (send_if_not (
-                    std::make_shared<Message>(o, protocol::mtMANIFESTS),
-                    peer_in_set (skip)));
+                auto const skip = hashRouter.shouldRelay (hash);
+                if(skip)
```

Comment:
This is a change in behavior. It looks like the old code if `prepRelay` returned false (not `shouldRelay`), then skip would be empty and we should relay to all the peers. In the new code, we don't relay to any peer. Is this intentional? It certainly fits a function named `shouldRelay` better.